src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -40,8 +40,16 @@ export interface VenueLandingPageProps {
   venueId?: string;
 }
 
+enum VenueAccessMode {
+  open = "open",
+  email = "email",
+  password = "password",
+  ticket = "ticket",
+  parent = "parent",
+}
```

Comment:
Guessing this was put here as a placeholder until https://github.com/sparkletown/sparkle/pull/1016 is merged?

Reply:
Yes, that's correct.